### ISSUE TITLE
Making Table Data from Blank DB file?

### ISSUE THREAD

**Issue by NIk0l41** [NIk0l41/GMCampaignManager#1]
1. Create `data.db`
2. Execute SQLite commands on `data.db`

Does this create an SQLite database?

**Comment by NIk0l41** [comment]
Ah ha! Instead of trying to create a db file using `System.IO`, we simply need to open a connection with the database.
```cs
SQLiteConnection con = new SQLiteConnection(conPath);
            con.Open();
```
is equivalent to
```sqlite
CREATE DATABASE filename;
```
So there we go, no need to create a file *then* open it.

**Comment by NIk0l41** [comment]
Fixed it here: 438c57b0bd31dd689da102495510dbb81c89e48c
